pkg/cmd/issue/shared/lookup_test.go

```diff
@@ -2,240 +2,94 @@ package shared
-				baseRepoFn: nil,
-			},
-			httpStub: func(r *httpmock.Registry) {
-				r.Register(
-					httpmock.GraphQL(`query IssueByNumber\b`),
-					httpmock.StringResponse(`{"data":{"repository":{
-						"hasIssuesEnabled": true,
-						"issue":{"number":13}
-					}}}`))
+			behavior: "when given URLs in different repos, errors",
+			args: []string{
+				"https://github.com/OWNER/REPO/issues/1",
+				"https://github.com/OWNER/OTHERREPO/issues/2",
 			},
-			wantIssue: 13,
-			wantRepo:  "https://example.org/OWNER/REPO",
+			expectedErr: true,
 		},
 		{
-			name: "project cards permission issue",
-			args: args{
-				selector:   "https://example.org/OWNER/REPO/issues/13",
-				baseRepoFn: nil,
-			},
-			httpStub: func(r *httpmock.Registry) {
-				r.Register(
-					httpmock.GraphQL(`query IssueByNumber\b`),
-					httpmock.StringResponse(`
-					{
-						"data": {
-							"repository": {
-								"hasIssuesEnabled": true,
-								"issue": {
-									"number": 13,
-									"projectCards": {
-										"nodes": [
-											null,
-											{
-												"project": {"name": "myproject"},
-												"column": {"name": "To Do"}
-											},
-											null,
-											{
-												"project": {"name": "other project"},
-												"column": null
-											}
-										]
-									}
-								}
-							}
-						},
-						"errors": [
-							{
-								"type": "FORBIDDEN",
-								"message": "Resource not accessible by integration",
-								"path": ["repository", "issue", "projectCards", "nodes", 0]
-							},
-							{
-								"type": "FORBIDDEN",
-								"message": "Resource not accessible by integration",
-								"path": ["repository", "issue", "projectCards", "nodes", 2]
-							}
-						]
-					}`))
-			},
-			wantErr:      true,
-			wantIssue:    13,
-			wantProjects: "myproject, other project",
-			wantRepo:     "https://example.org/OWNER/REPO",
+			behavior:    "when given an unparseable argument, errors",
+			args:        []string{"://"},
+			expectedErr: true,
 		},
 		{
-			name: "projects permission issue",
-			args: args{
-				selector:   "https://example.org/OWNER/REPO/issues/13",
-				baseRepoFn: nil,
-			},
-			httpStub: func(r *httpmock.Registry) {
-				r.Register(
-					httpmock.GraphQL(`query IssueByNumber\b`),
-					httpmock.StringResponse(`
-					{
-						"data": {
-							"repository": {
-								"hasIssuesEnabled": true,
-								"issue": {
-									"number": 13,
-									"projectCards": {
-										"nodes": null,
-										"totalCount": 0
-									}
-								}
-							}
-						},
-						"errors": [
-							{
-								"type": "FORBIDDEN",
-								"message": "Resource not accessible by integration",
-								"path": ["repository", "issue", "projectCards", "nodes"]
-							}
-						]
-					}`))
-			},
-			wantErr:      true,
-			wantIssue:    13,
-			wantProjects: "",
-			wantRepo:     "https://example.org/OWNER/REPO",
+			behavior:    "when given a URL that isn't an issue or PR url, errors",
```

Comment:
we need one more test case to verify inputs like `/cli/cli/issues/999` cause error (`url.Parse` parses that successfully).